core/src/main/java/com/scalar/db/api/DistributedStorageAdmin.java

```diff
@@ -52,7 +54,26 @@ public interface DistributedStorageAdmin extends Admin, AutoCloseable {
    * @throws ExecutionException if the operation fails
    * @return import table metadata in the ScalarDB format
    */
-  TableMetadata getImportTableMetadata(String namespace, String table) throws ExecutionException;
+  default TableMetadata getImportTableMetadata(String namespace, String table)
+      throws ExecutionException {
+    return getImportTableMetadata(namespace, table, Collections.emptyMap());
+  }
+
+  /**
+   * Get import table metadata in the ScalarDB format.
+   *
+   * @param namespace namespace name of import table
+   * @param table import table name
+   * @param overrideColumnsType a map of column data type by column name. Only set the column for
+   *     which you want to override the default data type mapping.
+   * @throws IllegalArgumentException if the table does not exist
+   * @throws IllegalStateException if the table does not meet the requirement of ScalarDB table
+   * @throws ExecutionException if the operation fails
+   * @return import table metadata in the ScalarDB format
+   */
+  TableMetadata getImportTableMetadata(
+      String namespace, String table, Map<String, DataType> overrideColumnsType)
+      throws ExecutionException;
```

Comment:
Add the `Map<String, DataType> overrideColumnsType` to override the type for columns in the `Admin.getImportTableMetadata(...)` method.